op-service/event/system.go

```diff
@@ -80,6 +92,34 @@ type systemActor struct {
 	emitPriority Priority
 }
 
+func (r *systemActor) traceAndLogEventEmitted(ctx context.Context, ev Event) context.Context {
+	_, path, line, _ := runtime.Caller(2)
```

Comment:
Can you please explain why we need this distinction? Any comments added to the code would be helpful.

Reply:
This is necessary only to log where we are emitting the event from, for example `loc=sequencing/sequencer.go:288` -- this is nice-to-have, as sometimes we emit the same event at multiple places, and it helps to know while debugging what exactly emitted the event.